WordPress/Classes/ViewRelated/Themes/ThemeBrowser.storyboard

```diff
@@ -142,17 +142,17 @@
                                             <rect key="frame" x="0.0" y="0.0" width="600" height="162"/>
                                             <subviews>
                                                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="1gq-o6-8td">
-                                                    <rect key="frame" x="0.0" y="0.0" width="600" height="107"/>
+                                                    <rect key="frame" x="0.0" y="0.0" width="600" height="108"/>
                                                     <subviews>
                                                         <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="mll-1k-nOP" userLabel="Header Divider Line">
-                                                            <rect key="frame" x="16" y="8" width="568" height="91"/>
+                                                            <rect key="frame" x="16" y="8" width="568" height="92"/>
                                                             <color key="backgroundColor" red="0.84705883260000003" green="0.88627451660000001" blue="0.91372549530000002" alpha="1" colorSpace="calibratedRGB"/>
                                                         </view>
                                                         <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" distribution="fillEqually" spacing="1" translatesAutoresizingMaskIntoConstraints="NO" id="U27-29-c8Q">
-                                                            <rect key="frame" x="0.0" y="0.0" width="600" height="107"/>
+                                                            <rect key="frame" x="0.0" y="0.0" width="600" height="108"/>
                                                             <subviews>
                                                                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Jwo-3t-dLB">
-                                                                    <rect key="frame" x="0.0" y="0.0" width="600" height="53"/>
+                                                                    <rect key="frame" x="0.0" y="0.0" width="600" height="53.5"/>
```

Comment:
Is the half point height here intended? I know with retina it's entirely acceptable, just want to double check ;)